tools/test_utils.py

```diff
@@ -154,6 +154,10 @@ def run_ngtf_pytests_from_artifacts(artifacts_dir):
     # Next run the ngraph-tensorflow python tests
     command_executor(["pip", "install", "-U", "pytest"])
     command_executor(["pip", "install", "-U", "psutil"])
+
+    test_manifest_file = TestEnv.get_test_manifest_filename()
+    os.environ['NGRAPH_TF_TEST_MANIFEST'] = test_manifest_file
+
```

Comment:
Please don't clobber the environment variable here. This can also be pushed down to where the manifest file is actually being used.

Reply:
yes, it was a bad miss. Fixing it.